locale/en_US/default.po

```diff
@@ -140,5 +140,7 @@ msgstr "R"
 msgid "default.contextSettings.emailSignature"
 msgstr ""
 "<br/>\n"
-"________________________________________________________________________<br/>\n"
-"<a href=\"{$contextUrl}\">{$contextName}</a>"
+"<hr /><br/>\n"
+"<p class='emaiSignatureContextUrlStyle'><a href=\"{$contextUrl}\">{$contextName}</a></p>"
+"<p class='emaiSignatureMailingAddressStyle'>{$mailingAddress}</p>"
+"<p class='emaiSignatureContactMailToStyle'><a href='mailto: {$contactEmail}'>{$contactName}</a></p>"
```

Comment:
`contactName` needs to be escaped.

Reply:
Also I think I preferred @NateWr's [suggestion](https://github.com/pkp/pkp-lib/issues/5049#issuecomment-548471676) for the syntax, where the email wasn't just linked but also displayed. It's necessary to mouseover the link to find the email when it's presented like this.

Reply:
Changed that to `<p class='emailSignatureContactMailToStyle'>{$contactName} <a href='mailto: {$contactEmail}'>{$contactEmail}</a></p>`

Reply:
Can you add a comma? So it appears like:

> Nate Wright, <a href="#">email@email.com</a>

Reply:
Added the comma